lib/mysql/table.js

```diff
@@ -27,7 +27,7 @@ module.exports = class Table extends _Table {
       try {
         let values = [];
   
-        let query = `SELECT COUNT(*) FROM ${this.name}`;
+        let query = `SELECT COUNT(*) AS notes_count FROM ${this.name}`;
```

Comment:
@dsmaldore We should make this more generic so that it can be used by any table for modularity. Why was it necessary to rename this? Could we name in count (lowercase) or ${this.name}_count?